strawberryfields/backends/base.py

```diff
@@ -478,6 +479,24 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
+        print("HI")
```

Comment:
lol don't forget to remove this before merging

Reply:
:thinking: hmm, thought I had already removed that